README.md

```diff
@@ -313,6 +313,18 @@ Using the default hatch configuration, dev dependencies listed in your
 from the lockfile you must remove them from the `default` environment
 on your `pyproject.toml` / `hatch.toml` file.
 
+### Disabling Changes to the Lockfile
+
+In some scenarios, like in CI/CD you want the tool to manage the installation
+of an environment, but you don't want the lockfile to be or changed without
+knowing. If you set the `PIP_COMPILE_DISABLED` environment variable to
```

Comment:
I don't have any good arguments for this right now but for some reason PIP_COMPILE_DISABLE seems like a more natural name to me than PIP_COMPILE_DISABLED

Reply:
And one letter less too! Works for me.